crates/gpui/src/platform/blade/blade_renderer.rs

```diff
@@ -331,6 +333,17 @@ impl BladeRenderer {
         window: &I,
         config: BladeSurfaceConfig,
     ) -> anyhow::Result<Self> {
+        // workaround for https://github.com/zed-industries/zed/issues/26143
+        let sample_count = std::env::var("ZED_SAMPLE_COUNT")
```

Comment:
Let's actually make either environment variable work.

Reply:
Now both `ZED_SAMPLE_COUNT` and `ZED_PATH_SAMPLE_COUNT` work.